RELEASE-NOTES.txt

```diff
@@ -1,6 +1,14 @@
 Unreleased
 ---
 
+1.81.1
+---
+* [*] List block v2: Fix text color inconsistencies with list items [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5096]
+* [*] Use default placeholder text color for native List Item [https://github.com/WordPress/gutenberg/pull/43353]
+* [*] Add BlockListCompact [https://github.com/WordPress/gutenberg/pull/43431]
```

Comment:
Nitpick: I noticed that this entry in the React Native editor's changelog has two `**` ([reference](https://github.com/WordPress/gutenberg/pull/43589/files#diff-d69739058d3a853b3b1346be9c94e43f82b7013cf8f8c10d8eea6b0d9e3f98acR17)).

Reply:
Since the [release tag](https://github.com/wordpress-mobile/gutenberg-mobile/releases/tag/v1.81.1) was already created, I'd update this in a different PR.

Reply:
Oops! 😅  I'll update it once I start merging back to `trunk` 👍